src/partition/repart.c

```diff
@@ -6698,6 +6780,20 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return r;
 
+        r = context_minimize(context);
```

Comment:
maybe rename context_minimize() → context_minimize_file_systems() or so?

Reply:
I moved the verity stuff into context_minimize() instead of adding a new function instead.